recipe/meta.yaml

```
@@ -56,87 +51,72 @@ outputs:
         - ninja
         - libgrpc
         - libprotobuf
+        - libgoogle-cloud-devel =={{ version }}
       host:
         - libabseil
-        - libcrc32c
-        - libcurl
         - libgrpc
         - libprotobuf
```

`libgoogle-cloud-all` will IMO need to pull in all the outputs from google-cloud-cpp-core:
```suggestion
        - libprotobuf
      run:
        - libgoogle-cloud-bigtable
        # not yet available
        # - libgoogle-cloud-compute
        # - libgoogle-cloud-compute-ai
        - libgoogle-cloud-iam
        - libgoogle-cloud-oauth2
        - libgoogle-cloud-policytroubleshooter
        - libgoogle-cloud-pubsub
        - libgoogle-cloud-spanner
        - libgoogle-cloud-storage
```

---

Fixed.